other/materials_designer/create_supercell.ipynb

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
<p>We can set supercell on each lines as square</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>